spanner/spring-data/src/main/com/example/spanner/SpannerRepositorySample.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * A quick start code for Spring Data Cloud Spanner.
+ * It demonstrates how to use a SpannerRepository to execute read-write queries
+ * generated from interface definitions.
+ *
+ */
+// [START spring_data_spanner_repository_sample]
+@SpringBootApplication
+public class SpannerRepositorySample {
```

Comment:
I don't think it needs to be a `@SpringBootApplication`. You can just annotate it as `@Component`.
You can even just keep the single method in the top level application.

Reply:
fixed.